docs/schemas/AgInspectionReport.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$id": "https://w3id.org/traceability/schemas/AgInspectionReport.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$comment": "{\"term\": \"AgInspectionReport\", \"@id\": \"https://w3id.org/traceability/AgInspectionReport\"}",
+    "title": "Agriculture Inspection Report",
+    "description": "Information on the Inspection and the observations made",
+    "type": "object",
+    "properties": {
+      "@context": {
+        "type": "array"
+      },
+      "type": {
+          "oneOf": [
+              { "type": "string" },
+              { "type": "array"}
+          ]
+      },
+      "facilityID": {
+        "$comment": "{\"term\": \"identifier\", \"@id\": \"https://schema.org/identifier\"}",
+        "title": "Facility Identifier",
+        "description": "Identifiers for an inspection facility.",
+        "type": "string"
+      },
+      "inspectorID": {
+        "$comment": "{\"term\": \"identifier\", \"@id\": \"https://schema.org/identifier\"}",
+        "title": "Inspector Identifier",
+        "description": "String of identifiers for inspectors.",
+        "type": "array"
+      },
+      "shipmentID": {
+        "$comment": "{\"term\": \"identifier\", \"@id\": \"https://schema.org/identifier\"}",
+        "title": "Shipment Identifier",
+        "description": "Identifiers for a shipment of goods.",
+        "type": "string"
+      },
+      "applicantID": {
+        "$comment": "{\"term\": \"identifier\", \"@id\": \"https://schema.org/identifier\"}",
+        "title": "Applicant Identifier",
+        "description": "Identifiers for the entity inspection applicant.",
+        "type": "string"
+      }, 
+      "inspectionLocation": {
```

Comment:
wondering if this would be better as an inspection object, with a `location` property.

Reply:
Agreed, I think a `location` property will be better here